sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -1016,11 +1016,13 @@ private void sendErrorAttachment(UUID errorId, Iterable<ErrorAttachmentLog> atta
                 if (attachment != null) {
                     attachment.setId(UUID.randomUUID());
                     attachment.setErrorId(errorId);
-                    if (attachment.isValid()) {
+                    if (!attachment.isValid()) {
+                        AppCenterLog.error(LOG_TAG, "Not all required fields are present in ErrorAttachmentLog.");
+                    } else if (attachment.getData().length > ErrorAttachmentLog.MAX_SIZE) {
+                        AppCenterLog.error(LOG_TAG, "Discarding attachment with size above " + ErrorAttachmentLog.MAX_SIZE + " bytes.");
```

Comment:
Can you add a size of the binary to the log message? This would be an evidence when we have a customer who says "it failed to send small size of binary".

Reply:
Maybe also attachment file name?

Reply:
To help customer troubleshoot which attachment they need to fix?

Reply:
sure